velox/exec/tests/TableScanTest.cpp

```diff
@@ -39,6 +39,22 @@ using namespace facebook::velox::common::test;
 using namespace facebook::velox::exec::test;
 
 namespace {
+std::string makeCastSql(const variant& v, const TypePtr& type) {
+  std::ostringstream out;
+  out << "CAST('";
+  if (type->isDate()) {
+    // Date variant should be created from string.
```

Comment:
Actually, DATE variant should be created from an int32_t because DATE time is backed by an INTEGER.

Reply:
Updated.